lib/v2/jin10/topic.js

```diff
@@ -0,0 +1,34 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const currentUrl = `https://xnews.jin10.com/topic/${ctx.params.id}`;
+    const response = await got(currentUrl);
+    const $ = cheerio.load(response.data);
```

Comment:
API is available: `https://reference-api.jin10.com/topic/getById?id=${id}`

Reply:
https://reference-api.jin10.com/topic/getById?id=396 returns `HTTP ERROR 502`

Reply:
https://reference-api.jin10.com/topic/getById?id=396 returns `HTTP ERROR 502`.
Do I need to set some secret keys? I couldn't find the docs.

Reply:
You need to set `x-app-id` and `x-version` header.

Reply:
I couldn't find what value to set for `x-app-id`. May I ask where you found that the API is called?

Reply:
https://www.gcores.com/collections, F12 and observe

Reply:
Never mind, I found it here: https://github.com/bafenzhiyimiao/foreign_dart_9/blob/fd0f4809ec78d00f5906a8647727818ef82deaa0/lib/choice/visual_screen.dart#L74﻿